assets/component-card.css

```diff
@@ -215,12 +215,12 @@
 }
 
 .card-information quantity-popover volume-pricing {
-  margin-top: 4.2rem;
+  margin-top: 0;
 }
```

Comment:
I think we can delete this now? `margin-top: 0;`?

Im still seeing this https://screenshot.click/09-52-dc3f7-yrdz9.png

Reply:
What about doing this? https://github.com/Shopify/dawn/pull/3408#discussion_r1558148104